### ISSUE TITLE
Error running win_launcher.exe

### ISSUE THREAD

**Issue by nadavshemesh** [nadavshemesh/silver-hacks#2]
Hi
![Error sct admin](https://user-images.githubusercontent.com/113446337/190474169-784c78bb-495b-4d0d-a489-dd25fdc36822.jpg)

I followed the instructions step by step but the program I downloaded ( https://github.com/nadavshemesh/silver-reversing-sct/blob/master/sct.exe) on the desktop gives me error. Above the image. I tried admin privileges, compatibility, disabling antivirus, reinstalling runtime and dll but nothing. Suggestions?

Ps

also this tool https://github.com/nadavshemesh/silver-hacks/blob/master/win_launcher.exe doesn't work. missing dlls but I assure you I installed them one by one.

Sigh help plz

thx

_Originally posted by @MisterMax82 in https://github.com/nadavshemesh/silver-hacks/issues/1#issuecomment-1246671981_

**Comment by nadavshemesh** [comment]
I tried to add a dependancy the executable has (cygwin1.dll), maybe that could help address this issue, 
please redownload/ clone [the repository](https://github.com/nadavshemesh/silver-hacks) and try again. 
let me know if it helped

**Comment by MisterMax82** [comment]
Dependancy are ok, is no longer requested !!!
Now , my problem (I think) is my pc ( error 0xc000007b windows 10 pro) with both programs : https://github.com/nadavshemesh/silver-reversing-sct/blob/master/sct.exe and https://github.com/nadavshemesh/silver-hacks/blob/master/win_launcher.exe.
Do you recommend that I format to solve the problem or do you have a second solution? I have really tried everything
thx

**Comment by nadavshemesh** [comment]
No i dont think formating is nessicary, perhaps you should try and build it on your own machine.
to try and build it youself, follow these steps:
1. Google "gcc for windows", or just use this [link](https://dev.to/gamegods3/how-to-install-gcc-in-windows-10-the-easier-way-422j) I found using google search with instructions on how to install gcc on a windows os. (I havn't tried this guide myself)
2. After gcc is installed, open up the commandline and navigate to the repository's main directory (cd /path/to/repo)
3. Run this command to build:
```
gcc win_launcher.cpp -lstdc++ -o .\win_launcher.exe
```
and then you can try and run it using 
```
.\win_launcher.exe
```


with regards to [the .sct tool](https://github.com/nadavshemesh/silver-reversing-sct)
you can build it in the same way. after installing gcc, navigate to its directory and run:
```
gcc -I .\ .\utils.c .\sct\sct.c .\sct\structure\object.c .\sct\structure\pattern.c .\sct\print\asm\console.c .\sct\game\gamefunc.c .\sct\convert\convert.c .\sct\convert\asm\sctasm.c .\sct\convert\bin\sctbin.c .\file\fileutils.c .\sct\print\bin\console.c .\sct\game\gamevar.c .\catalog.c .\sct\print\bin\file.c .\sct\print\asm\file.c -o .\sct.exe
```

I Hope this helps

**Comment by MisterMax82** [comment]
HAPPYYYYYYYYYYYYYYYYYYY
FINALLY
ha ha playing with grandfather . Too strong. Now I will try with Gorgon.
thank you very much BRO
If you like old games like me, I tried to make a virtua tennis (first) cheat table. Sure there will be mistakes however it is fun. I unfortunately am not very experienced but something I managed to do. thanks again brother.
Too nice grandpa ^_^
![grandpa](https://user-images.githubusercontent.com/113446337/190903058-ff53e3d3-cddc-4698-a845-af2ad456c35e.jpg)
![function](https://user-images.githubusercontent.com/113446337/190903074-42ab0041-0862-4022-9d51-4ae510b1f960.jpg)
